src/YoutubeAtomSticky.tsx

```diff
@@ -104,6 +117,8 @@ type Props = {
     children: JSX.Element;
 };
 
+const isMobile = detectMobile();
```

Comment:
Should we include `tablet:true` here so that we detect for mobile *and* tablet given that we want to detect devices with (primarily) touch screens

https://github.com/juliangruber/is-mobile#:~:text=To%20add%20support%20for%20tablets%2C%20set%20tablet%3A%20true.

Reply:
ah yes - i just merged so will make a new PR to cover that, really well spotted!